Cargo.toml

```diff
@@ -33,6 +33,7 @@ linkerd2-timeout   = { path = "lib/timeout" }
 
 linkerd2-proxy-api = { git = "https://github.com/linkerd/linkerd2-proxy-api", tag = "v0.1.8" }
 
+byteorder = "1.3.1"
```

Comment:
The `byteorder` lib may be removed from Linkerd if we implement a new and dedicated Kafka Rust codec. If we do need such a lib in Linkerd, then is it correct to put it here?

Reply:
Yes, if we need to add an additional dependency, this is the correct place to put it.

However, it's worth noting that we're already using the [`bytes` crate](https://docs.rs/crate/bytes/0.4.12), which also provides functions for reading bytes in big- and little-endian orders (I believe it uses `byteorder` internally). So, I think we may be able to implement the Kafka protocol detection using functions from the `bytes` crate, without adding a new dependency. If possible, that would probably be preferable.